src/main/resources/application.yml

```diff
@@ -133,39 +133,39 @@ akhq:
   security:
     default-group: admin
     groups:
-    - name: admin
-      roles:
-      - topic/read
-      - topic/insert
-      - topic/delete
-      - topic/config/update
-      - node/read
-      - node/config/update
-      - topic/data/read
-      - topic/data/insert
-      - topic/data/delete
-      - group/read
-      - group/delete
-      - group/offsets/update
-      - registry/read
-      - registry/insert
-      - registry/update
-      - registry/delete
-      - registry/version/delete
-      - acls/read
-      - connect/read
-      - connect/insert
-      - connect/update
-      - connect/delete
-      - connect/state/update
-    - name: reader
-      roles:
-      - topic/read
-      - node/read
-      - topic/data/read
-      - group/read
-      - registry/read
-      - acls/read
-      - connect/read
-    - name: no-roles
-      roles: []
+      admin:
```

Comment:
Just a minor changes here : 
```yaml
admin: 
  name: Admin
   roles: 
```
And everywhere else. 

We need to have a name & a key here because people with LDAP have group case sensitive. 
I have a lot of issues about that in the past that lead me to go to array (that was not a good idea at the end).

Reply:
I added a name property to the default groups with the same name as the key. I also changed the PostConstruct of `SecurityProperties` so that, when a name is not specified in the configuration, the key will be used as default.